lib/puppet/provider/cisco_command_config/cisco.rb

```diff
@@ -85,4 +85,35 @@ def command=(cmds)
     info "Successfully updated:\n#{e.previous.join("\n")}" unless e.previous.empty?
     raise
   end # command=
+
+  def test_get
+    # This method is for beaker use only. It allows beaker to retrieve any
+    # configuration it needs from the device using puppet resource. Callers
+    # must pass a filter string to test_get.
+    # Example usage:
+    #  puppet resource cisco_command_config 'cc' test_get='incl feature'
+    cmd = 'show running-config all'
+    cmd << " | #{@resource[:test_get]}" if @resource[:test_get]
+
+    result = @node.get(command: cmd)
+    "\n" + result unless result.nil?
```

Comment:
I think it would be helpful to add a debug here as well to dump out `result`.  Useful for troubleshooting.